src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1243,15 +1243,46 @@
         combined-list (vec (concat base-ext-list csr-ext-list))]
     (ensure-ext-list-has-cn-san subject combined-list)))
 
+(defn report-successful-cert-signing
+  "Log message and report to the activity service if available about successful cert signing."
+  [report-activity-or-nil message subject certname ip-address]
+  (let [commit {:service {:id "puppetserver"}
```

Comment:
I think the service id should probably be "puppet-ca" which is what we name the CA's trapperkeeper service.